Source/WebCore/page/Quirks.cpp

```diff
@@ -2996,15 +2981,23 @@ static void handleHBOMaxQuirks(QuirksData& quirksData, const URL& quirksURL, con
         return;
 
     auto topDocumentHost = quirksURL.host();
-    if (topDocumentHost != "play.hbomax.com"_s)
-        return;
+    if (topDocumentHost == "play.hbomax.com"_s) {
+        quirksData.enableQuirks({
+            // play.hbomax.com https://bugs.webkit.org/show_bug.cgi?id=244737
+            QuirksData::SiteSpecificQuirk::ShouldEnableFontLoadingAPIQuirk,
+#if HAVE(PIP_SKIP_PREROLL)
+            // play.hbomax.com rdar://158430821
+            QuirksData::SiteSpecificQuirk::ShouldDisableAdSkippingInPip,
+#endif
+        });
+    }
 
     quirksData.enableQuirks({
-        // play.hbomax.com https://bugs.webkit.org/show_bug.cgi?id=244737
-        QuirksData::SiteSpecificQuirk::ShouldEnableFontLoadingAPIQuirk,
-#if HAVE(PIP_SKIP_PREROLL)
-        // play.hbomax.com rdar://158430821
-        QuirksData::SiteSpecificQuirk::ShouldDisableAdSkippingInPip,
+#if ENABLE(DESKTOP_CONTENT_MODE_QUIRKS)
```

Comment:
It seems this should be on the outside of `quirksData.enableQuirks` to avoid method calls in places that don't need it.

Reply:
I moved some of the things around. Taken into account @stwrt comment too.